joplin/base/wagtail_hooks.py

```diff
@@ -108,6 +108,9 @@ def media(self):
             js.append(webpack_loader_utils.get_files('janisBranchSettings')[0]['url'])
             return forms.Media(css=css, js=js)
 
+        def is_shown(self, request):
+            return request.user.is_superuser
+
```

Comment:
I know branch settings options are only shown if we are running locally / review branches, but i went ahead and hid the menu item if we were logged in as editors